dev/kyuubi-tpcds/src/main/scala/org/apache/kyuubi/tpcds/benchmark/Benchmark.scala

```diff
@@ -17,13 +17,6 @@
 
 package org.apache.kyuubi.tpcds.benchmark
 
-import scala.concurrent._
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.duration.DurationInt
-import scala.language.implicitConversions
-import scala.util.{Failure => SFailure, Success, Try}
-import scala.util.control.NonFatal
```

Comment:
`UnusedImports` are removed by spotless

Reply:
@Yikf it's not unused imports, and it breaks the CI.

Reply:
could you please send a follow-up PR to recover the build? otherwise, I need to revert this one.

Reply:
FYI: you can use `./build/mvn clean package -Ptpcds -pl dev/kyuubi-tpcds -am` to verify if it works